R/treeData.R

```diff
@@ -118,7 +118,7 @@ setMethod("rowTreeData", signature = c(x = "TreeSummarizedExperiment"),
 DEFAULT_TREE_DATA_COLS <- c("parent","node","branch.length","label")
 .clean_tree_data <- function(tree_data){
     tree_data %>%
-        select(DEFAULT_TREE_DATA_COLS)
+        select(all_of(DEFAULT_TREE_DATA_COLS))
```

Comment:
Was there a bug or why this has changed?

Reply:
Not a bug, but tidyselect 1.2.0 has started to require it